jenkins_jobs/trigger_nightly_host_os_build.groovy

```diff
@@ -69,9 +68,8 @@ job('trigger_nightly_host_os_build') {
     downstreamParameterized {
       trigger('build_host_os_iso') {
         block {
-          buildStepFailure('FAILURE')
-          failure('FAILURE')
-          unstable('UNSTABLE')
+          buildStepFailure('UNSTABLE')
+          failure('UNSTABLE')
```

Comment:
Those names indeed do not say much. From the Jenkins interface, you can see:
`buildStepFailure('UNSTABLE')` means "Fail this build step if the triggered build is worse or equal to UNSTABLE"
and `failure('UNSTABLE')` means "Mark this build as failure if the triggered build is worse or equal to UNSTABLE"